pkgs/top-level/all-packages.nix

```diff
@@ -3440,6 +3440,8 @@ with pkgs;
 
   azure-cli = callPackage ../tools/admin/azure-cli { };
 
+  azure-cli-extensions = recurseIntoAttrs azure-cli.extensions;
```

Comment:
Shouldn't it use `pkgs/by-name` now ?

Reply:
You mean `azure-cli` in general should be in `pkgs/by-name`, or `azure-cli-extensions` specifically? I'm not sure how you would archive the latter one without moving all of azure-cli into by-name, and the extensions would still need an entry in all-packages.nix, wouldn't they?

Thanks for your review. :)